source/v2/openapi.yaml

```diff
@@ -1226,6 +1226,43 @@ paths:
           lang: shell
           source: |-
             curl -X GET https://www.trade-tariff.service.gov.uk/api/v2/geographical_areas/countries
+
+### Changes
+  /changes:
+    get:
+      summary: Retrieves a list of goods nomenclature objects that have changed since the previous day
+      description: |
+        Retrieves a list of goods nomenclature objects that have changed since the previous day
+      tags:
+      - Changes
+      parameters:
+        - name: 'as_of'
+          in: path
+          required: false
+          default: current date
+          description: |
+            Retrieve changes as they existed between the `as_of` date and the previous day. Use the format `YYYY-MM-DDD`. While this is not a required field it is advised to include it with all requests even
```

Comment:
I wonder if we should give the path parameter a different name in the documentation? I figure on_date or equivalent